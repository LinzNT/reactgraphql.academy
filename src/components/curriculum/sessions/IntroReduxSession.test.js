import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import IntroReduxSession from './IntroReduxSession'

describe('<IntroReduxSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<IntroReduxSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
