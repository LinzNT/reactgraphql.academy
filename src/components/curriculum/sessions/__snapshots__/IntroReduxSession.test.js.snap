// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroReduxSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Introduction to functional programming
      <styled.ul>
        <styled.li>
          Data and behaviour
        </styled.li>
        <styled.li>
          Data in, data out
        </styled.li>
        <styled.li>
          Mutations
        </styled.li>
        <styled.li>
          Pure functions
        </styled.li>
      </styled.ul>
    </styled.li>
    <styled.li>
      Redux Principles
      <styled.ul>
        <styled.li>
          Store, Reducers, Actions
        </styled.li>
        <styled.li>
          Unique source of truth
        </styled.li>
      </styled.ul>
    </styled.li>
    <styled.li>
      React-Redux: Provider and Connect
    </styled.li>
    <styled.li>
      Build your own version of
      <code>
        redux
      </code>
       and
      <code>
        react-redux
      </code>
       using TDD
    </styled.li>
    <styled.li>
      Fetching data from the server to Redux
    </styled.li>
    <styled.li>
      Configure Redux from scratch in a React app
    </styled.li>
  </styled.ul>
</Session>
`;
